configs/karaf/islandora_id_mapping_service.script

```diff
@@ -1,4 +1,4 @@
-feature:repo-add mvn:edu.amherst.acdc/acrepo-karaf/1.0.6/xml/features
+feature:repo-add mvn:edu.amherst.acdc/acrepo-karaf/1.0.8/xml/features
 feature:install acrepo-services-activemq
```

Comment:
This has been replaced with fcrepo-service-activemq.  If this script gets run after alpaca, we'll be fine to delete this.